src/sentry/eventstore/__init__.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import
+
+from django.conf import settings
+
+from sentry.utils.services import LazyServiceWrapper
+
+from .base import EventStorage  # NOQA
+
+backend = LazyServiceWrapper(
```

Comment:
@tkaemming, I think you have context on how we expose services in the Sentry code base. Is there any reason to expose eventstore by getting the implementation from configuration? I think this is very useful when we have multiple implementations of a service or where we expect to have multiple implementations. But I am not sure this would be the case. I think we could easily hardcode the specific implementation here.

Reply:
We've hard coded things in the past, and then later made them configurable when necessary. I think that hardcoding here is fine — it's easy enough to change later, if we decide to.

Reply:
Ok, will update